packages/storybook-addon-theme/__mocks__/index.js

```diff
@@ -0,0 +1,6 @@
+var React = require('react')
+
+const MockTheme = props => React.createElement(React.Fragment, props)
```

Comment:
what is your thinking on why it's desirable to mock the theme in stories?

Reply:
I like the snapshots to be smaller and more inline with what is actually being tested. having the theme components in each snapshot adds a lot of extra levels to sort through